apps/prairielearn/elements/pl-drawing/mechanicsObjects.js

```diff
@@ -3023,12 +3023,12 @@ mechanicsObjects.byType['pl-paired-vector'] = class extends PLDrawingBaseElement
       'selectable',
       'evented',
     ];
-    var options1 = {};
-    var options2 = {};
-    for (var i = 0; i < includedAttributes.length; i++) {
-      if (typeof options[includedAttributes[i]] != typeof options['abcnonsense']) {
```

Comment:
I don't know why this was written this way, but the new code should be basically equivalent and is much clearer. There's a possibility that we relied on the distinction between "property does not exist" and "property exists and is undefined".